contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -743,6 +743,10 @@ Incorporating the default values into the `Spec` ensures that `Spec` depicts the
 full desired state so that it is easier for the system to determine how to
 achieve the state, and for the user to know what to anticipate.
 
+Default values can be specified on a field using the `+default=` tag. Primitives
+will have their values directly assigned while structs will go through the
+JSON unmarshalling process.
```

Comment:
Maybe include an example of a semi-complex json form. Is there something specific in the kubebuilder book about the format of these that can be interesting? Also we need to mention that non-pointer fields can't default to anything but the zero value.

Reply:
Updated to include the non-pointer comment. I think for the json example, it would be more suited in the website documentation rather than here?